firmware/quadruna/VC/src/tasks.c

```diff
@@ -468,7 +476,7 @@ _Noreturn void tasks_runCanRx(void)
     {
         CanMsg rx_msg;
         io_can_popRxMsgFromQueue(&rx_msg);
-        io_telemMessage_broadcast(&rx_msg);
+        //        io_telemMessage_broadcast(&rx_msg);
```

Comment:
is this desired?

Reply:
yes. Telem messages are clogging the rxtask on vc